jena/JosekiExtensions/ivy.xml

```diff
@@ -79,7 +79,10 @@
 		<dependency org="org.junit.jupiter" name="junit-jupiter-engine" rev="5.1.0" conf="test->default"/>
 		<dependency org="org.junit.platform" name="junit-platform-runner" rev="1.1.0" conf="test->default"/>
 
-		<exclude org="commons-logging"/>
+    <!-- https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind -->
+    <dependency org="com.fasterxml.jackson.core" name="jackson-databind" rev="2.9.7"/>
```

Comment:
- [ ] Please add a conf attribute to this line.  If the library is used in unit tests, it should be "test->default", otherwise "bridge->default".